src/components/DatePicker/DatePickInput/DatePickInput.tsx

```diff
@@ -120,40 +113,22 @@ const DatePickInput = React.forwardRef<HTMLInputElement, DatePickInputProps>(
 
     const renderBase = () => {
```

Comment:
i feel that there is no need for `renderBase` anymore. We can just use `if` to return the end result